docs/development/core/development-functional-tests.asciidoc

```diff
@@ -80,7 +87,8 @@ node scripts/functional_test_runner --exclude-tag skipCloud
 
 When run without any arguments the `FunctionalTestRunner` automatically loads the configuration in the standard location, but you can override that behavior with the `--config` flag. List configs with multiple --config arguments.
 
-* `--config test/functional/config.js` starts Elasticsearch and Kibana servers with the selenium tests configuration.
+* `--config test/functional/config.js` starts Elasticsearch and Kibana servers with the WebDriver tests configured to run in Chrome
+* `--config test/functional/config.firefox.js` starts Elasticsearch and Kibana servers with the WebDriver tests configured to run in Firefox
 * `--config test/api_integration/config.js` starts Elasticsearch and Kibana servers with the api integration tests configuration.
 
```

Comment:
@LeeDr I added a short description about how to run x-pack tests https://github.com/elastic/kibana/pull/41806/commits/18335e8c5e19c236056dcee4405d2debd7186597

I don't think we need to be more precise since it really depends on what tests are executed